libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2247,6 +2249,11 @@ MAV_RESULT GCS_MAVLINK::handle_command_set_message_interval(const mavlink_comman
     return MAV_RESULT_FAILED;
 }
 
+MAV_RESULT GCS::set_message_interval(uint8_t channel, uint32_t msg_id, int32_t interval_us)
+{
```

Comment:
this probably needs a sanity check that the channel is < MAVLINK_COMM_NUM_BUFFERS but this could wait until after we have a consensus.

Reply:
fixed it